Week4/Lecture7/Assignment2/assignment2/src/main/java/lecture7/assignment2/service/EmployeeService.java

```diff
@@ -0,0 +1,39 @@
+package lecture7.assignment2.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmployeeService {
+
+    private EmailService emailService;  
+
+    // Constructor Injection
+    @Autowired
```

Comment:
Can we remove @Autowired in this case, and explain?

Reply:
Yes, after some research if a class has only one constructor, Spring will automatically use that constructor for dependency injection, even if the @Autowired annotation is not present.